examples/tutorials/api/irfs.py

```diff
@@ -0,0 +1,458 @@
+
+        migra_min = coords["migra"][:, :-1, :]
+        migra_max = coords["migra"][:, 1:, :]
+
+        # Analytical formula for integral of Gaussian
+        s = np.sqrt(2) * sigma
+        t1 = (migra_max - 1 - bias) / s
+        t2 = (migra_min - 1 - bias) / s
+        pdf = (scipy.special.erf(t1) - scipy.special.erf(t2)) / 2
+        pdf = pdf / (migra_max - migra_min)
+
+        r1 = np.rollaxis(pdf, -1, 1)
+        r2 = np.rollaxis(r1, 0, -1)
+        data = r2 * np.ones(axes.shape)
+
+        data[data < pdf_threshold] = 0
+
+        return cls(
+            axes=axes,
+            data=data.value,
+        )
+
+
+# Make a test case
+energy_axis_true = MapAxis.from_energy_bounds(
+    "0.1 TeV", "100 TeV", nbin=3, name="energy_true"
+)
+
+migra_axis = MapAxis.from_bounds(0, 4, nbin=2, node_type="edges", name="migra")
+
+fov_lon_axis = MapAxis.from_edges([-1.5, -0.5, 0.5, 1.5] * u.deg, name="fov_lon")
+fov_lat_axis = MapAxis.from_edges([-1.5, -0.5, 0.5, 1.5] * u.deg, name="fov_lat")
+
+data = np.array(
+    [
+        [
+            [
+                [4.99582964e-01, 4.99582964e-01, 4.99582964e-01],
+                [4.99582964e-01, 4.99582964e-01, 4.99582964e-01],
+                [4.99582964e-01, 4.99582964e-01, 4.99582964e-01],
+            ],
+            [
+                [2.06259405e-04, 2.06259405e-04, 2.06259405e-04],
+                [2.06259405e-04, 2.06259405e-04, 2.06259405e-04],
+                [2.06259405e-04, 2.06259405e-04, 2.06259405e-04],
+            ],
+        ],
+        [
+            [
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+            ],
+            [
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+            ],
+        ],
+        [
+            [
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+                [5.00000000e-01, 5.00000000e-01, 5.00000000e-01],
+            ],
+            [
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+                [0.00000000e00, 0.00000000e00, 0.00000000e00],
+            ],
+        ],
+    ]
+)
+
+
+edisp3d = EnergyDispersion3D(
+    [energy_axis_true, migra_axis, fov_lon_axis, fov_lat_axis], data=data
+)
+
+print(edisp3d)
+
+energy = [1, 2] * u.TeV
+migra = np.array([0.98, 0.97, 0.7])
+fov_lon = [0.1, 1.5] * u.deg
+fov_lat = [0.0, 0.3] * u.deg
+
+edisp3d.evaluate(
+    energy_true=energy.reshape(-1, 1, 1, 1),
+    migra=migra.reshape(1, -1, 1, 1),
+    fov_lon=fov_lon.reshape(1, 1, -1, 1),
+    fov_lat=fov_lat.reshape(1, 1, 1, -1),
+)[0][2]
```

Comment:
Can you explain the [0][2] to the user?
Can you precise also how the interpolations over the axis and the data are made? Or make a link towards an other tutorial...